tests/test_package.py

```diff
@@ -1,7 +1,4 @@
 import os
-import zipfile
-
-import pytest
```

Comment:
Is pytest not required for the `assert` style syntax below, and zipfile for the ZipPackage namespace?

Reply:
Not in this case, only a native Python assert statement is used and here it is running on Travis: https://travis-ci.org/openpreserve/fido/jobs/596043501#L455 This also means that the code is optimised away in some circumstances but its fine for testing.